src/pheval/utils/phenopacket_utils.py

```diff
@@ -92,6 +92,13 @@ class PhenopacketUtil:
     def __init__(self, phenopacket_contents: Phenopacket):
         self.phenopacket_contents = phenopacket_contents
 
+    def sample_id(self) -> str:
+        """Retrieve the sample ID."""
```

Comment:
Change the doc string to say """Retrieve the sample ID from a phenopacket or proband of a family."""